packages/xo-server-audit/src/index.js

```diff
@@ -290,12 +294,12 @@ class AuditXoPlugin {
     }
   }
 
-  async _getRecords({ id, ndjson = false }) {
+  async _getRecords({ id, maxRecords = 10e3, ndjson = false }) {
```

Comment:
Use a global variable for the max number.

Reply:
Would it be relevant to add `maxRecords` to the configuration?

Reply:
IMHO, the value is large enough for this quick fix and using a setting from `staticConfig` is not a simple replacement.

We'll improve this later if necessary.